File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 8/1/2025, 10:42:17 PM
class Solution {
    public int removeDuplicates(int[] nums) {
        int j=1;
        for(int i=1;i<nums.length;i++){
           if(nums[i]!=nums[i-1]){
            nums[j]=nums[i];
            j++;
           }
        }
       
        return j;
    }
}

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.c
// Last updated: 8/1/2025, 10:42:24 PM
int removeDuplicates(int* nums, int numsSize) {
    int i,un=0;
    for(i=1;i<numsSize;i++){
        if(nums[i]!=nums[un]){
        un++;
        nums[un]=nums[i];
        }
    }
return un+1;
}